src/ResourceManager/Compute/Commands.Compute/help/Set-AzureRMVMDataDisk.md

```diff
@@ -114,6 +114,20 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -StorageAccountType
+The virtual machine managed disk's account type.```yaml
```

Comment:
@hyonholee please make sure the ` ```yaml` string is on the next line by itself. There is a bug in `platyPS` that is causing this string to be at the end of the parameter's description rather than on a line by itself.